modules/Company/CompanyCore/DTO/CreateCompanyDTO.php

```diff
@@ -8,14 +8,15 @@ class CreateCompanyDTO
 {
     public function __construct(
         public string $name,
+        public string $user_name,
         private string $email,
+        private string $serial_no,
         private string $phone,
         private string $country_id,
         private string $company_type_id,
         private string $company_field_id,
         private string $registration_type_id,
         private string $registration_no,
```

Comment:
Please follow this pattern with variables
```suggestion
        private string $companyTypeId,
        private string $companyFieldId,
        private string $registrationTypeId,
        private string $registrationNo,
```